ddtrace/_logger.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from logging.handlers import RotatingFileHandler
+import os
+
+from ddtrace.internal.utils.formats import asbool
+
+
+DEFAULT_FILE_SIZE_BYTES = 15 << 20  # 15 MB
+
+
+def configure_ddtrace_logger():
```

Comment:
```suggestion
def configure_ddtrace_file_logger():
```

Just to be slightly more precise